graphene-pore/graphene-pore.ipynb

```diff
@@ -0,0 +1,183 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import mbuild as mb\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "from foyer import Forcefield\n",
+    "from calc_numberdensity import calc_number_density"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "water = mb.load('files/tip3p.mol2')\n",
+    "water.name = 'SOL'"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "c_ff = Forcefield('files/carbon.xml')\n",
+    "spce = Forcefield('files/spce.xml')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "try:\n",
+    "    GraphenePoreSolvent = mb.recipes.GraphenePoreSolvent\n",
+    "except ImportError:\n",
```

Comment:
I think we should try to catch `AttributeError` here as well. I hit a case of having `mb.recipes` installed but not with `GraphenePoreSolvent` as an attribute.

Reply:
Good idea, this _should_ be handled now.